packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
+                              <atomic-result-date
+                                class="slack-result-date"
+                                field="date"
+                                format="h:mm A - MM/DD/YYYY"
+                              ></atomic-result-date>
+                            </span>
+                          </div>
+                          <div class="slack-result-msg-excerpt">
+                            <atomic-result-section-excerpt>
+                              <atomic-result-text field="excerpt"> </atomic-result-text>
+                            </atomic-result-section-excerpt>
+                          </div>
+                          <atomic-field-condition must-match-slackmessagetype="Normal">
+                            <div class="slack-result-normal-top"></div>
+                          </atomic-field-condition>
+                          <atomic-field-condition must-match-slackmessagetype="ThreadStart">
+                            <div class="slack-result-thread-top">
+                              <atomic-result-section-bottom-metadata>
+                                <atomic-result-fields-list>
+                                  <div class="slack-result-thread-bottom-text">
+                                    <span
+                                      ><atomic-result-number field="slackmessagereplycount"></atomic-result-number
+                                    ></span>
+                                    <atomic-field-condition must-match-slackmessagereplycount="1">
+                                      <span><atomic-text value="reply"></atomic-text></span>
+                                    </atomic-field-condition>
+                                    <atomic-field-condition must-not-match-slackmessagereplycount="1">
+                                      <span><atomic-text value="replies"></atomic-text></span>
+                                    </atomic-field-condition>
+                                  </div>
+                                </atomic-result-fields-list>
+                              </atomic-result-section-bottom-metadata>
+                            </div>
+                          </atomic-field-condition>
+                        </div>
+                      </atomic-result-section-title>
+                    </div>
+                  </atomic-field-condition>
+                  <atomic-field-condition if-not-defined="slackmessagetype">
+                    <div class="slack-title-icon">
+                      <atomic-result-section-visual class="slack-atomic-result-icon" image-size="icon">
+                        <atomic-result-icon>
+                          <img
+                            loading="lazy"
+                            src="https://static.cloud.coveo.com/searchui/v2.10096/image/slack.svg"
+                            class="thumbnail"
+                          />
+                        </atomic-result-icon>
```

Comment:
Instead of an `img` for the default icon, I recommend that you use `atomic-icon` for SVG icons.

```suggestion
                        <atomic-result-icon>
                          <atomic-icon
                            icon="https://static.cloud.coveo.com/searchui/v2.10096/image/slack.svg"
                          />
                        </atomic-result-icon>
```

Reply:
beware: `icon="assets://slack"` isn't equivalent, since the "slack" icon in our assets is colorless.